packages/eslint-config-custom/react.js

```diff
@@ -0,0 +1,86 @@
+module.exports = {
+  root: true,
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+  },
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  ignorePatterns: ['dist', '.eslintrc.cjs', 'prettier.config.cjs'],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:unicorn/recommended',
+    'plugin:react/recommended',
+    'plugin:react/jsx-runtime',
+    'plugin:react-hooks/recommended',
+  ],
+  plugins: ['@typescript-eslint', 'react'],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: ['./tsconfig.json', './tsconfig.node.json'],
+  },
+  rules: {
+    'no-console': 2,
+    eqeqeq: 2,
+  },
+  overrides: [
+    {
+      files: ['**/*.{jsx,ts,tsx}'],
+      rules: {
+        'react/jsx-pascal-case': 2,
```

Comment:
I wrote it in the [discussion page](https://github.com/monstar-lab-oss/ml-frontend/discussions/1201#discussioncomment-8637129), but I would like to discuss whether setting the severity to 2 (error) is appropriate. 

It may be beneficial, especially for beginners, to enforce best practice, although it may become an annoyance to more experienced developers. Of course, it will vary on the rule, so I'd like to discuss with this in mind.

Regarding naming conventions, it may be enough to set it to 1 (warning).

Reply:
@ptrkdan Normal syntax errors such as quotes and semicolons do not have a big impact on the normal working flow. I think these errors should be in the form of `warning`. For errors that affect the entire team's work flow, such as file name and export method, I think we should set level to `error`.